nemo_skills/inference/generate.py

```diff
@@ -367,12 +384,21 @@ def fill_prompt(self, data_point, data):
         )
 
     def llm_generate(self, data_points, data, is_async=False):
-        generation_params = {
-            "prompts": [self.fill_prompt(dp, data) for dp in data_points],
-            "stop_phrases": combine_stop_phrases(self.prompt.stop_phrases, self.extra_stop_phrases),
-            **asdict(self.cfg.inference),
-            **self.extra_generate_params,
-        }
+
+        if self.cfg.prompt_format == "openai":
```

Comment:
is this change needed? Shouldn't the original code work fine since the fill_prompt will just return the data_point?